instrumentation-genai/opentelemetry-instrumentation-vertexai-v2/tests/test_gemini.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+from vertexai.preview.generative_models import GenerativeModel, Part
+
+# from opentelemetry.semconv_ai import SpanAttributes
+
+
+@pytest.mark.vcr
+def test_vertexai_generate_content(exporter):
```

Comment:
maybe start with plain chat tests from openai (just convert them) I think vision/multi-modal could be better discussed in aggregate across the two?

Reply:
Will update tomorrow